data-prepper-plugins/s3-source/build.gradle

```diff
@@ -31,11 +31,6 @@ dependencies {
     testImplementation 'com.github.tomakehurst:wiremock:3.0.0-beta-5'
     testImplementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
     testImplementation project(':data-prepper-test-common')
-    testImplementation project(':data-prepper-plugins:parquet-codecs')
```

Comment:
I think we need to include the new codecs. My previous comment was only to remove parquet and avro.

Thus, you should add:

```
testImplementation project(':data-prepper-plugins:csv-processor')
testImplementation project(':data-prepper-plugins:parse-json-processor')
testImplementation project(':data-prepper-plugins:newline-codecs')
```